score.go

```diff
@@ -0,0 +1,952 @@
+package pubsub
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/libp2p/go-libp2p-core/protocol"
+
+	ma "github.com/multiformats/go-multiaddr"
+)
+
+type PeerScoreParams struct {
```

Comment:
Can we move the two Params structs to a `score_params.go` file?

Reply:
ok...

Reply:
done.